test/e2e/specs/editor/editor__schedule.ts

```diff
@@ -75,27 +84,24 @@ describe( DataHelper.createSuiteTitle( `Editor: Schedule` ), function () {
 			await editorPage.closeAllPanels();
 		} );
 
-		it( `View post as ${ accountName }`, async function () {
-			const testPage = await browser.newPage();
+		it( `View post as the author`, async function () {
+			const tmpPage = await context.newPage(); // Calling from context opens new tab (same session)
 
-			const testAccount = new TestAccount( accountName );
-			await testAccount.authenticate( testPage );
+			await tmpPage.goto( postURL.href );
+			await new PublishedPostPage( tmpPage ).validateTextInPost( postContent );
 
-			await testPage.goto( postURL.href );
-			const publishedPostPage = new PublishedPostPage( testPage );
-			await publishedPostPage.validateTextInPost( postContent );
-			await testPage.close();
+			await tmpPage.close();
```

Comment:
**question**: If I recall our conversation correctly, this is how we work around the caching issue affecting Atomic sites, correct?

Would it make sense to call `context = await browser.newContext();` in this test step? I'm thinking about the principle of declaring/initializing values close to where it's used instead of all the way at the top. What do you think about it?